main/signers/hot/HotSigner/worker.js

```diff
@@ -56,9 +56,10 @@ class HotSignerWorker {
     pseudoCallback(null, addHexPrefix(hex))
   }
 
-  signTypedData (key, params, pseudoCallback) {
+  signTypedData ({ key, typedMessage }, pseudoCallback) {
```

Comment:
unless there's a specific reason let's keep the key and the typed message as separate parameters as they're not logically linked in any way

Reply:
I can't remember my thinking on this specifically, but I suspect it was simply to group and have fewer params.  I'll separate them again and see if there was any other reason.

Reply:
Ok so the answer is below - previous eth-sig-util had them separate, new version has everything together, I expect I was just following this

```
const signature = ethSigUtil.signTypedMessage(key, { data: params.typedData }, params.version)
```

```
const signature = signTypedData({ privateKey: key, data, version })
```